apis/v1alpha1/cluster_types.go

```diff
@@ -145,6 +145,9 @@ type CrdbClusterStatus struct {
 	// CrdbContainerImage is the container that will be installed
 	// +operator-sdk:csv:customresourcedefinitions:type=status, displayName="CrdbContainerImage",xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
 	CrdbContainerImage string `json:"crdbcontainerimage,omitempty"`
+	// SQLHost is the host to be used with SQL ingress
+	// +operator-sdk:csv:customresourcedefinitions:type=status, displayName="SQLHost",xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
+	SQLHost string `json:"sqlHost,omitempty"`
```

Comment:
Apologies if I'm missing the obvious, but why do we need this as well as the host value in the Ingress definition?

Reply:
We need to detect if the SQL host is changed or not. So to keep track of old SQL host and new SQL host (if updated in CR), we are setting current the current host in status

Reply:
Is it possible to pull the host from the Ingress object? We don't store any other analogous information on the status - for instance, instead of storing num CRDB nodes on the status, we pull it directly from the StatefulSet.

If this is not possible, then do we also need to store the host for the UI ingress here?

Reply:
I followed similar approach of `cluster.Spec().Image.Name` and `cluster.Status().CrdbContainerImage`. Though both contains the same info but it's used to take a decision to force trigger Version checker.

Similarly, spec and status SQL host can change during update and we need some differentiator to trigger certificate regeneration before actually going to  `ExposeIngressAction` action.

This is not applicable for UI Ingress because change in UI ingress host does not trigger any other actor. `ExposeIngressAction` will take care of updating the UI ingress Host